.github/workflows/changelog.yml

```diff
@@ -18,26 +20,28 @@ jobs:
         uses: release-drafter/release-drafter@v5.15.0
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-      # Generates a YAML changelog file using https://github.com/jenkinsci/jenkins-core-changelog-generator
-      - name: Generate YAML changelog draft
-        id: jenkins-core-changelog-generator
-        uses: jenkinsci/jenkins-core-changelog-generator@master
-        env:
-          GITHUB_AUTH: github-actions:${{ secrets.GITHUB_TOKEN }}
-      - name: Upload Changelog YAML
-        uses: actions/upload-artifact@v2.2.4
```

Comment:
I would rather prefer to keep it for now until the flow is fully settled. I use this asset in one of my experiments (changleog => opengraph generator)